src/components/search/Hit.tsx

```diff
@@ -74,7 +79,27 @@ export default function HitComponent({ hit }: { hit: SearchHit }): JSX.Element {
             </Flex>
           )}
         </Text>
+        <Button
+          rounded="md"
+          w="full"
+          mt={4}
+          py="7"
+          bg={useColorModeValue("blue.900", "blue.50")}
+          color={useColorModeValue("white", "gray.900")}
+          _hover={{
+            transform: "translateY(2px)",
```

Comment:
```suggestion
            transform: "translateY(-2px)",
```

i think that might look better